CONTRIBUTING.md

```diff
@@ -25,6 +25,9 @@ gulp
 
 When you're ready, continue with next steps:
 
+- Run ```sh
+gulp uglify
```

Comment:
I fixed this file: [CONTRIBUTING.md](https://github.com/fdaciuk/ajax/blob/master/CONTRIBUTING.md) :smile: